docs/cmd/flux.md

```diff
@@ -69,25 +72,25 @@ Command line utility for assembling Kubernetes CD pipelines the GitOps way.
 ```
       --context string      kubernetes context to use
   -h, --help                help for flux
-      --kubeconfig string   path to the kubeconfig file (default "~/.kube/config")
+      --kubeconfig string   absolute path to the kubeconfig file
   -n, --namespace string    the namespace scope for this operation (default "flux-system")
       --timeout duration    timeout for this operation (default 5m0s)
       --verbose             print generated objects
 ```
 
 ### SEE ALSO
 
-* [flux bootstrap](flux_bootstrap.md)	 - Bootstrap toolkit components
-* [flux check](flux_check.md)	 - Check requirements and installation
-* [flux completion](flux_completion.md)	 - Generates completion scripts for various shells
-* [flux create](flux_create.md)	 - Create or update sources and resources
-* [flux delete](flux_delete.md)	 - Delete sources and resources
-* [flux export](flux_export.md)	 - Export resources in YAML format
-* [flux get](flux_get.md)	 - Get the resources and their status
-* [flux install](flux_install.md)	 - Install or upgrade Flux
-* [flux logs](flux_logs.md)	 - Display formatted logs for Flux components
-* [flux reconcile](flux_reconcile.md)	 - Reconcile sources and resources
-* [flux resume](flux_resume.md)	 - Resume suspended resources
-* [flux suspend](flux_suspend.md)	 - Suspend resources
-* [flux uninstall](flux_uninstall.md)	 - Uninstall Flux and its custom resource definitions
+* [flux bootstrap](/cmd/flux/flux_bootstrap/)	 - Bootstrap toolkit components
```

Comment:
Doesn't this break the links in mkdocs?

Reply:
All links are broken when you click on them from GitHub UI, I guess mkdocs fails the same.

Reply:
I tested this with `mkdocs serve` and it works fine, Hugo is unable to parse `.md` links without them being wrapped in [`ref` or `relref`](https://gohugo.io/content-management/cross-references/).

Reply:
Yes, how about `[flux bootstrap](/cmd/flux_bootstrap/)`? That should work both in `mkdocs` and `hugo` land.

Reply:
https://www.docsy.dev/docs/best-practices/site-guidance/#linking

@alisondy and I talked about this as well.

Reply:
Ah, that is what it was supposed to be. Let me push a fix in a couple minutes.

Reply:
@dholbach @hiddeco do you think is ok to have broken links when people read the docs with GitHub UI? I expect users to read the docs on our website, so maybe this is acceptable.

Reply:
I see no other way at the moment, we may opt to use the `ref` thing once everything has been moved over so that they are more obviously broken.